temba/flows/tests.py

```diff
@@ -1988,6 +1988,91 @@ def test_flow_keyword_create(self):
         flow = Flow.objects.get(name='Survey Flow')
         self.assertEqual(flow.triggers.all().count(), 0)
 
+    def test_flow_create_with_ussd_trigger(self):
+        self.login(self.admin)
+
+        post_data = dict()
+        post_data['name'] = "USSD Flow 1"
+        post_data['ussd_trigger'] = "notallowed"
+        post_data['flow_type'] = Flow.USSD
+        response = self.client.post(reverse('flows.flow_create'), post_data, follow=True)
+
+        self.assertEqual(response.context_data['form'].errors,
+                         {u'ussd_trigger': [u'USSD code must contain only *,# and numbers']})
```

Comment:
am guessing these are leftovers from copying from the terminal but you don't need `u''` prefixes